### ISSUE TITLE
Android: wind direction?

### ISSUE THREAD

**Issue by kpagcha** [erikflowers/weather-icons#176]
I am trying to use the wind direction icons, but they all share the same content element (`f0b1`), so how can I include them in my `strings.xml` file?

**Comment by erikflowers** [comment]
It is meant to use CSS transforms to rotate them. No idea how you would rotate them in a native app. Just apply a transform to the degree you need it? like 90 degrees would go from pointing upwards, to pointing to the right. But I have no idea how it works, that's just a guess.

**Comment by kpagcha** [comment]
Yeah I have no idea how to do that with Android. Is it possible to create individual icons for the wind directions instead of using CSS on them? I am sure many Android devs use this icon pack. It'd be awesome.

**Comment by kpagcha** [comment]
Alright I figured out how to do it, using rotate animations with a duration of 0 (default):

```java
RotateAnimation rotate = new RotateAnimation(0f, 360f - deg, Animation.RELATIVE_TO_SELF, 0.5f, Animation.RELATIVE_TO_SELF, 0.5f);
rotate.setFillAfter(true);
windIcon.setAnimation(rotate);
windIcon.setIcon(getActivity().getString(R.string.wind_icon));
```

**Comment by olkunmustafa** [comment]
Hi @kpagcha 

I couldn't show icons in my Android app even though I write unicode text in my strings.xml files it is shown as "f07d". how did you solve this problem ?

**Comment by Sujal1** [comment]
@olkunmustafa In your strings.xml, rewrite it as **&#xf07d**
